test/models/UserTypeCheckModelSpec.scala

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import base.SpecBase
+import org.scalatestplus.mockito.MockitoSugar
+import play.api.libs.json.Json
+
+
+class UserTypeCheckModelSpec extends SpecBase with MockitoSugar with DataHelper {
+  val largeProducerUserAnswersNoContractPacking: UserAnswers = UserAnswers("id", Json.obj("howManyLitresGlobally" -> "large",
+    "howManyImports" -> Json.obj("lowBand" -> 1,"highBand" -> 1)))
+  val smallProducerUserAnswersNoOperatePackagingSites: UserAnswers = UserAnswers("id", Json.obj("howManyLitresGlobally" -> "small",
+    "howManyContractPacking" -> Json.obj("lowBand" -> 1,"highBand" -> 1), "thirdPartyPackagers" -> true,
+    "howManyImports" -> Json.obj("lowBand" -> 1,"highBand" -> 1)))
+  val notAProducerUserAnswersNoThirdPartyPackagers: UserAnswers = UserAnswers("id", Json.obj("howManyLitresGlobally" -> "xnot",
+    "howManyOperatePackagingSites" -> Json.obj("lowBand" -> 1,"highBand" -> 1), "howManyContractPacking" -> Json.obj("lowBand" -> 1,"highBand" -> 1)))
+  val answersNoImports: UserAnswers = UserAnswers("id", Json.obj("howManyLitresGlobally" -> "xnot",
+    "howManyOperatePackagingSites" -> Json.obj("lowBand" -> 1,"highBand" -> 1), "thirdPartyPackagers" -> false))
+
+  "UserTypeCheckModel" - {
+    "isLarge returns true if HowManyLitresGloballyPage was answered with > 1 million" in {
```

Comment:
all reference to pages in your test names can be the object instead with string interpolation (in case the page gets removed or renamed to protect these tests, such as
```suggestion
    s"isLarge returns true if $HowManyLitresGloballyPage was answered with > 1 million" in {
```
would be cool to amend all of the test names so they are future proofed

Reply:
Completed